packages/studio/src/components/HighlightingContainer.tsx

```diff
@@ -64,12 +81,12 @@ class HighlightingClass extends Component<HighlightingProps> {
     if (!childNode) {
       return;
     }
-    const rect = rectToJson(childNode.getBoundingClientRect());
-    if (!isEqual(this.props.rect, rect)) {
-      this.props.setRect(rect);
-    }
     if (this.props.activeComponentUUID !== this.props.uuid) {
+      const rect = rectToJson(childNode.getBoundingClientRect());
       this.props.setActiveUUID(this.props.uuid);
+      this.props.clearSelectedComponents();
+      this.props.addSelectedComponentUUID(this.props.uuid);
```

Comment:
I think lines 87 and 88 are now internally handled by `setActiveComponentUUID` (line 86)?

Reply:
Removed